docs/source/export.rst

```diff
@@ -39,28 +39,27 @@ serialized.
 
     ExportedProgram:
         class GraphModule(torch.nn.Module):
-            def forward(self, arg0_1: f32[10, 10], arg1_1: f32[10, 10]):
+            def forward(self, x: "f32[10, 10]", y: "f32[10, 10]"):
                 # code: a = torch.sin(x)
-                sin: f32[10, 10] = torch.ops.aten.sin.default(arg0_1);
+                sin: "f32[10, 10]" = torch.ops.aten.sin.default(x);  x = None
```

Comment:
nit: I would prefer to remove the stuff after the semicolon :P

Reply:
I can do that. I kept them here because I saw in the section of `Export for Training and Inference` we have kept the contents after the semicolon. Shall we also remove those?

Also, would users be confused if they see the print-out contents are not exactly matching the doc here?